source/administration/config-settings.rst

```diff
@@ -112,6 +123,42 @@ Enable Team Directory ``"EnableTeamListing": false``
 
 ________
 
+Policy (Enterprise)
+``````````````````````````
+Settings to configure the permission restrictions for sending team invite links and managing channels.  
+
+Enable sending team invites from ``"RestrictTeamInvite": "all"``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Restrict the permission levels required to send team invites. Note: if permissions are restricted and "Get Team Invite Link" is used to share a link, it will need to be regenerated after the desired users joined the team.
+
+``all``: Selecting "All team members" allows any team member to invite others using an email invitation or team invite link.
+
+``admin``: Selecting "Team and System Admins" hides the email invitation and team invite link in the Main Menu from users who are not Team or System Admins.
```

Comment:
Propose

``admin``: Only Team and System Admins can invite others via **Main Menu** > **Invite New Member** or **Main Menu** > **Get Team Invite Link**. The options are hidden from other users.